### ISSUE TITLE
Signup page: fix tab order on form elements

### ISSUE THREAD

**Issue by secretrobotron** [mozilla/id.webmaker.org#146]
The tab order on the signup page form elements should be go from Password :arrow_forward: Checkbox :arrow_forward: Button
